helm_deploy/README.md

```diff
@@ -1,32 +1,25 @@
 
-### Example test deploy command
-
-```
-helm --namespace digital-prison-services-dev --tiller-namespace digital-prison-services-dev upgrade notm ./notm/ --install --values=values-dev.yaml --values=secrets-example.yaml --dry-run --debug
-```
-
-Test template output:
-
+### Example deploy command
 ```
-helm template ./notm/ --values=values-dev.yaml --values=secrets-example.yaml
+helm --namespace keyworker-api-dev  --tiller-namespace keyworker-api-dev upgrade keyworker-api ./keyworker-api/ --install --values=values-dev.yaml --values=example-secrets.yaml
 ```
 
 ### Rolling back a release
 Find the revision number for the deployment you want to roll back:
 ```
-helm --tiller-namespace digital-prison-services-dev history notm -o yaml
+helm --tiller-namespace keyworker-api-dev history keyworker-api -o yaml
 ```
```

Comment:
This seems like the wrong app name... Unless I've completely misunderstood.

Reply:
oops wrong file, good spot